crypto/x509/x509_vfy.c

```diff
@@ -3364,6 +3364,9 @@ STACK_OF(X509) *X509_build_chain(X509 *target, STACK_OF(X509) *certs,
     return result;
 }
 
+/* note that there's a corresponding minbits_table in ssl/ssl_cert.c
```

Comment:
The format of the comment is wrong. It should start with `/*` on a separate line.